.github/workflows/ci-multiarch.yaml

```diff
@@ -65,7 +65,7 @@ jobs:
           cache-from: type=registry,ref=${{ env.CACHE_IMAGE_FULL }}
           cache-to: type=registry,ref=${{ env.CACHE_IMAGE_FULL }},mode=max
           context: .
-          file: ./apache.Dockerfile
+          file: ./dockerfiles/apache.Dockerfile
```

Comment:
good idea to move it into a subfolder. Why call it apache.Dockerfile instead of just **Dockerfile**? 

Also seems like most projects are using a `build/dockerfiles/` folder

* https://github.com/eclipse/che-jwtproxy/tree/master/build/dockerfiles
* https://github.com/eclipse-che/che-machine-exec/tree/main/build/dockerfiles
* https://github.com/eclipse-che/che-devfile-registry/tree/master/build/dockerfiles

Reply:
Originally the dashboard was using the build folder for something else and it was gitignored. In the new changes the dashboard outputs their build files to /lib (like che-theia does) so we can use build/dockerfiles for our dockerfiles :smile: